SU-SWS/stanford_capx
# 8.x 1.x port

# NOT READY FOR REVIEW 
_but ready for feedback_

# Summary
This is a start to set up the module scaffolding and put in the correct routes, dependencies, and start stubbing out the authentication form.

# Needed By (Date)

# Criticality

# Steps to Test
- Checkout this branch `git checkout 8.x-1.x-port`
- Navigate to `admin/config/capx/authentication`
- Verify pages exist as expected

# Affects 

# Associated Issues and/or People
## Related JIRA ticket(s)

## Related PRs

## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
